ci-operator/step-registry/storage/conf/storageclass/install-gcp-hyperdisk/storage-conf-storageclass-install-gcp-hyperdisk-ref.yaml

```
@@ -0,0 +1,11 @@
+ref:
+  as: storage-conf-storageclass-install-gcp-hyperdisk
```

```suggestion
  as: storage-conf-storageclass-pre-set-default-gcp-hyperdisk
```
Maybe better, since the step changes the default storage class by day 0(before the `chain: ipi-install`). We could also do it as day2(after the `chain: ipi-install`) configuration by patch the csidriver and create the new storageclass. Use the `pre/post` maybe helpful to distinguish what phase the step used for.  WDYT?

---

i'm pro day0, I will update it to the suggested name. :+1: